### PR TITLE
Update README.md re. vgrid

### PR DESCRIPTION
**Pull request by aekiss**
See https://github.com/COSIMA/domain-tools/issues/33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Code and tools to edit and manipulate ocean model grids and topographies.
 
 Below is a list of included tools and short documentation for each.
 
+**Note:** in all cases `vgrid` is assumed to be a MOM5 vgrid (with $2n+1$ values for an $n$-level model). Using a MOM6 `vgrid` (with $n+1$ values) will produce incorrect results.
```
Comment (aekiss) [inline README.md]:
```suggestion
**Note:** in all cases `<vgrid>` is assumed to be a MOM5 vertical grid file (with $2n+1$ values for an $n$-level model). Using a MOM6 `<vgrid>` file (with $n+1$ values) will produce incorrect results.
```

**Review by micaeljtoliveira — APPROVED** [review]

**Comment by micaeljtoliveira** [comment]
@aekiss It should be easy to add an assertion in the Fortran code to check if the number of values is odd, as that should always be the case for MOM5 vgrids (2n+1 values for n levels). It wouldn't complain about MOM6 vgrids where n+1 is odd, but it's better than nothing... What do you think?

**Comment by aekiss** [comment]
Yep, might as well, but with a comment to explain it will only catch this problem half the time.
It won't help with the 51 vs 101 case we discussed yesterday, but would catch 76 vs 151 which will be the new default at all resolutions.

**Comment by micaeljtoliveira** [comment]
Merging this. Will implement the assertion in the Fortran code in a separate MR.

**Pull request merged**